artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/broker.xml

```diff
@@ -78,7 +78,7 @@ ${global-max-section}
          <!-- amqpLowCredits: The server will send the # credits specified at amqpCredits at this low mark -->
 
          <!-- Acceptor for every supported protocol -->
-         <acceptor name="artemis">tcp://${host}:${default.port}?tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;protocols=CORE,AMQP,STOMP,HORNETQ,MQTT,OPENWIRE;useEpoll=true;amqpCredits=1000;amqpLowCredits=300</acceptor>
+         <acceptor name="artemis">tcp://${host}:${default.port}?anycastPrefix=jms.queue.;multicastPrefix=jms.topic.;tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;protocols=CORE,AMQP,STOMP,HORNETQ,MQTT,OPENWIRE;useEpoll=true;amqpCredits=1000;amqpLowCredits=300</acceptor>
```

Comment:
shouldn't this be enabled manually by the user when needed? this will add back the old semantics by default. it seems a drawback to me?

Reply:
this feels wrong to me.. .@mtaylor

Reply:
@jbertram we could have this commented out... like.. if you have old acceptors. use this configuration...

but we can't have this by default.


We can change the compatiblity tests to add prefixes and make sure they work together.

Reply:
@jbertram  but isn't this changing the behaviour for old clients? say if you don't change the config.